test/functional/comment_controller_test.rb

```diff
@@ -42,6 +42,29 @@ def teardown
     assert_template partial: 'questions/_comment'
   end
 
+  test 'should create wiki comments' do
+    UserSession.create(rusers(:bob))
+    assert_difference 'Comment.count' do
```

Comment:
In a separate unit test, which I think is a good idea, we could also add a comment to the same record, but using back-end code instead of sending a `POST` request via the HTTP API. Unit tests are for more fundamental behaviors of classes, whereas functional tests are for the public facing HTTP interfaces. So unit tests tend to be more efficient to run - they don't need the whole server running to try out. Generally I think one test per public HTTP URL is good enough, and then more thorough testing in unit tests because it's so much more efficient to do it there. Once you know you can rely on your unit tests, you have the confidence that your classes are really predictable and stable, and you can just use them with less worry, make sense? Can you add a new checklist item to your main issue for a set of comment/wiki unit tests?